app/views/admin/widget/cards/_form.html.erb

```diff
@@ -1,14 +1,18 @@
-<%= form_for [:admin, @card] do |f| %>
+<%= render "admin/shared/globalize_locales", resource: @card %>
+
+<%= translatable_form_for [:admin, @card] do |f| %>
+
   <div class="small-12 medium-6">
-    <%= f.text_field :label %>
+    <%= f.translatable_text_field :label %>
   </div>
 
-  <%= f.text_field :title %>
+  <%= f.translatable_text_field :title %>
```

Comment:
I think we either need to add the label option like in banners, where we add `label: t("admin.banners.banner.title")`, or change something in translatable form in order to find the correct label automatically.

Reply:
Right! I did this a while ago by adding `label: @object.class.human_attribute_name(method),` to the form builder method, but I later removed it as I couldn't remember why it was necessary (it's so that the label is rendered as "Title" instead of "Title en"). 

I'll add it back.

Reply:
Fixed by 49b9ae7